hazelcast-enterprise-operator/hazelcastcluster.crd.yaml

```diff
@@ -1,11 +1,7 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: hazelcastenterprises.hazelcast.com
-  labels:
```

Comment:
Aren't these things necessary for testing? In the workflows I use label information.

Reply:
Afaik, release script just checking labels of pods not CRDs. CR/Pod have those labels already btw --> https://github.com/hazelcast/hazelcast-operator/blob/master/hazelcast-operator/hazelcast.yaml#L5-L8

Reply:
Now I took a look and you are right, I will delete these labels from `crd`s in my PR then.